src/index.html

```diff
@@ -11,7 +11,153 @@
+          >
+        </label>
+        <label class="personal__input" for="birthDate">Full date of birth:
+          <input
+          type="date"
+          id = "birthDate"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label for="privacy">I accept the term of the agreement
+          <input
+          type="checkbox"
+          id = "privacy"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="personal__info">
+        <legend>Registration:</legend>
+        <label class="personal__input" for="email">E-mail:
+          <input
+          type="email"
+          placeholder="email@example.com"
+          id="email"
+          autocomplete="off"
+          required
+          >
+        </label>
+        <label for="pass">Password:
+          <input
+          type="password"
+          id = "pass"
+          autocomplete="off"
+          required
+          >
+        </label>
+      </fieldset>
+      <fieldset class="personal__info">
+        <legend>An interesting fact about you?</legend>
+        <div class="personal__input">
+          <label>Do you love cats?</label>
+          <label for="yes">
+            <input
+            class="input__cats"
+            type="radio"
+            name="cats"
+            value="yes"
+            id="yes"
+            >
+            Yes
+          </label>
+          <label for="no">
+            <input
+            class="input__cats"
+            type="radio"
+            name="cats"
+            value="no"
+            id="no"
+            >
+            No
+          </label>
+        </div>
+        <label class="personal__input" for="color">What is your favorite color?
+          <input type="color" id="color">
+        </label>
+        <label class="personal__input" for="time">What time do you go to bed?
+          <input
+          type="time"
+          id="time"
+          step="1"
+          >
+        </label>
+        <label class = "personal__input" for="car">What are your favorite brands of car
+          <select
+          name="car"
+          id="car"
+          autocomplete="off"
+          multiple
+          >
+            <option>BMW</option>
+            <option>Audi</option>
+            <option>Lada</option>
+          </select>
+        </label>
+        <label for="rateOurWork">How do your rate our work?
+          <input
+          type="range"
+          id="rateOurWork"
+          min="1"
+          max="5"
+          step="1"
+          value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="personal__info">
+        <legend>Additional info:</legend>
+        <label class="personal__input" for="comment">
+          Comments
+          <textarea
+          name="comment"
+          id="comment"
+          >
+          </textarea>
+        </label>
+        <label for="recomendUs">Would you recommend us?
+          <select name="recomend" id = "recomendUs">
```

Comment:
```suggestion
          <select name="recomend" >
```
do you need `id` here?